### PR TITLE
Major update to SERVOx_FUNCTION page

### PR DESCRIPTION
**Pull request by Hwurzburg**
Changed page to apply to ALL vehicles and to allow logical path to find needed output functions
- grouped by vehicle or function
- noted for which vehicle(s) the function applies
- added explanations to links for every function

once merged, I intend to add a DEFAULT ASSIGNMENTS table for SERVO1-8 listing every vehicle type and what it assigns as defaults to each output...

### CODE REVIEW

**Comment by Hwurzburg** [comment]
went ahead and added it....

File: common/source/docs/common-rcoutput-mapping.rst [hunk 1]
```diff
@@ -1,51 +1,539 @@
...
+------------------------------
+
+In the advanced parameter view of your GCS you will find that each
+SERVO output channel has a SERVOn_FUNCTION parameter. For example,
+SERVO5_FUNCTION controls the output function of channel 5,
+SERVO6_FUNCTION controls the output function of channel 6 and so on.
 
-   The ability to remap multicopter motors is available in Copter-3.4 (and higher).
-   The parameters used to control the output channels has changed from RCx_FUNCTION to SERVOx_FUNCTION in Copter-3.5 and higher.
+Not all the functions are available in each vehicle. Defaults are set to 0 when firmware a vehicle type is first loaded. Choosing a frame
```
Comment (tridge) [inline common/source/docs/common-rcoutput-mapping.rst]: grammer? "when firmware a vehicle type"

Comment (Hwurzburg) [reply common/source/docs/common-rcoutput-mapping.rst]: fixed...thanks

File: common/source/docs/common-rcoutput-mapping.rst [hunk 2]
```diff
@@ -508,4 +508,32 @@ Sprayer Pump/Mixer
 
 These outputs are controlling a :ref:`sprayer` .
 
-
+DEFAULT VALUES
```
Comment (rmackay9) [inline common/source/docs/common-rcoutput-mapping.rst]: looking good, personally I'd prefer if we reserve all caps for parameters..

**Comment by rmackay9** [comment]
Merged, thanks!

**Comment by bnsgeyer** [comment]
You guys move quick.  I will follow up with another PR to add the dualheli to the table and take a look at other improvements for tradheli.  @Hwurzburg  thanks for including me on the review.  I appreciate it.

**Comment by Hwurzburg** [comment]
@bnsgeyer Bill, maybe you could take a look at eliminating the param naming error in the heli code  that causes these errors in the wiki build: 

copter/source/docs/parameters.rst:18469: WARNING: Duplicate explicit target name: "h_rsc_crv_enable__ap_motorsheli_rsc".

copter/source/docs/parameters.rst:18708: WARNING: Duplicate explicit target name: "h_rsc_gov_enable__ap_motorsheli_rsc".

thanks

**Comment by bnsgeyer** [comment]
@Hwurzburg  I have an [RSC PR](https://github.com/ArduPilot/ardupilot/pull/12069) that should resolve this error.

**Comment by vierfuffzig** [comment]
@Hwurzburg afaik functions 73 / 74 (throttle left / right) are not rover specific, the are used in plane too for differential thrust on twin motor setups.

and thanks for all your work on the wiki!

cheers, basti.

**Comment by Hwurzburg** [comment]
Thanks will update...

**Pull request merged**